### PR TITLE
[BACKLOG-8276] Allow disabling automatic updates to the view.

### PR DESCRIPTION
**Pull request by carlosrusso**
Allows disabling automatic updates to the view.
@pentaho/millenniumfalcon Please review.

### CODE REVIEW

**Comment by lucboudreau** [comment]
Please categorize and squash these commits appropriately before merging. Thanks.

**Comment by lucboudreau** [comment]
Please create tests for BitSet.

**Comment by carlosrusso** [comment]
@lucboudreau Thank you for the warning. We are using this PR as a means to share a diff.  
The branch to be eventually merged is currently a work-in-progress and the late modifications are a consequence of our review cycle.

If you believe this is generating too much noise being heard outside of millenniumfalcon, please let us know, and we will move the discussion to a fork of this repo.

**Comment by lucboudreau** [comment]
I don't mind the noise. Thanks for asking. :)

**Comment by carlosrusso** [comment]
Rebased on master and squashed commit history.

File: package-res/resources/web/pentaho/util/BitSet.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    constructor: function(bits) {
+      this.set(bits || 0);
+    },
+
+    /**
+     * Returns a value that indicates if there are no bits set.
+     *
+     * @return {boolean} `true` if no bits are set, `false` otherwise.
+     */
+    get isEmpty() {
```
Comment (davidmsantos90) [inline package-res/resources/web/pentaho/util/BitSet.js]: For getters and setters we are using a different doc pattern like this:

/**
  * Gets the value that indicates if there are no bits set.
  *
  * @type boolean
  * @readonly
  */

**Comment by dcleao** [comment]
@pentaho/millenniumfalcon please see this new version where I've removed the did:create concept altogether, and returned to something similar to what we had previously.

**Comment by dcleao** [comment]
Need to merge this now, cause others are dependent on having this by Monday.
Despite this, please make any comments and I will make any needed changes in a subsequent PR.
Thanks.

**Pull request merged**